visitor/urls.py

```diff
@@ -4,5 +4,5 @@
 
 urlpatterns = [
     url(r'^visitor/$', api.VisitorList.as_view()),
-    url(r'^visitor/(?P<pk>[0-9]+)/$', api.VisitorDetail.as_view())
+    url(r'^visitor/(?P<pk>[0-9]+)/$', api.VisitorDetail.as_view()),
```

Comment:
Please add relevant commit message in case you are adding other file apart from what you originally planned to commit.